packages/create-block/README.md

```diff
@@ -65,9 +65,9 @@ More examples:
 
 When you scaffold a block, you must provide at least a `slug` name, the `namespace` which usually corresponds to either the `theme` or `plugin` name, and the `category`. In most cases, we recommended pairing blocks with plugins rather than themes, because only using plugin ensures that all blocks still work when your theme changes.
 
-## Available Commands
+## Available Commands [ESNext template]
 
-Inside that bootstrapped directory _(it doesn't apply to `es5` template)_, you can run several commands:
+When bootstraped with the `esnext` templateyou can run several commands inside the directory:
```

Comment:
One more thing, `wpScriptsEnabled` enables those commands. Once we sort out the final shape of template definition we can update this section to reflect it.

Reply:
Yes the commands get enabled by `wpScriptsEnabled` but at the end it matters what the creator of the template has in their `scripts` in the `package.json`. 

Thats why I changed it to only refer to the `esnext` template.